src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/design/hideutilityclassconstructor/Example2.java

```diff
@@ -0,0 +1,48 @@
+/*xml
+<module name="Checker">
+  <module name="TreeWalker">
+    <module name="HideUtilityClassConstructor">
+      <property name="ignoreAnnotatedBy" value="SkipMe, java.lang.Deprecated" />
+    </module>
+   </module>
+</module>
+*/
+
+package com.puppycrawl.tools.checkstyle.checks.design.hideutilityclassconstructor;
+
+// xdoc section -- start
+class Example2 { // violation
+
+  public Example2() {
+
+  }
+
+  public static void fun() {
+
+  }
+}
+
+@SkipMe
+class IgnoredClass1 { // ok, ignored by annotation
```

Comment:
lets change `SkipMe` to `SpringBootApplication` to let user see exact example and it will help easily understand.